Expensify/App
# Add storybook and a Header story

### Details
Adds Storybook to E.cash

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/164021

### Tests
1. `npm i`
2. Run `npm run storybook`
3. Verify that storybook opens

### QA
1. Navigate to `https://staging.expensify.cash/docs`
1. Verify that storybook is accessible

### Tested On

-  Mobile Web
-  Desktop
-  iOS
-  Android

### Screenshots


#### Web
![2021-05-13_14-04-52](https://user-images.githubusercontent.com/32969087/118201868-35be7b00-b3f4-11eb-85fe-0c110f299384.png)
